src/components/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from "react";
```

Comment:
Rename this to index.tsx so the imports will be /components/dashboard

Breakdown this component into smaller components.